packages/web-api/src/WebClient.ts

```diff
@@ -42,7 +42,7 @@ export interface WebClientOptions {
   tls?: TLSOptions;
   timeout?: number;
   rejectRateLimitedCalls?: boolean;
-  headers?: Record<string, unknown>;
+  headers?: Record<string, string>;
```

Comment:
This change is related to upgrading to the latest axios, which I believe has updated (added?) its types. I assumed this would be an innocent change, since HTTP headers are always strings, but what does everyone else think?

Reply:
This looks fine.